packages/config/config/devices/0x019b/z-trm6.json

```diff
@@ -0,0 +1,469 @@
+			"maxValue": 400,
+			"defaultValue": 400,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "9",
+			"label": "External Sensor Max Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 400,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "10",
+			"label": "Internal Sensor Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unit": "0.1 °C",
+			"unsigned": false
+		},
+		{
+			"#": "11",
+			"label": "Floor Sensor Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unit": "0.1 °C",
+			"unsigned": false
+		},
+		{
+			"#": "12",
+			"label": "External Sensor Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unit": "0.1 °C",
+			"unsigned": false
+		},
+		{
+			"#": "13",
+			"label": "Regulation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Hysteresis",
+					"value": 0
+				},
+				{
+					"label": "PWM",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "14",
+			"label": "Temperature Control Hysteresis",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 30,
+			"defaultValue": 5,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "15",
+			"label": "Temperature Display",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Setpoint",
+					"value": 0
+				},
+				{
+					"label": "Measured",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "16",
+			"label": "Active Display Brightness",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unit": "10%",
+			"unsigned": true
+		},
+		{
+			"#": "17",
+			"label": "Standby Display Brightness",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 5,
+			"unit": "10%",
+			"unsigned": true
+		},
+		{
+			"#": "18",
+			"label": "Temperature Report Interval",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 65535,
+			"defaultValue": 840,
+			"unit": "seconds",
+			"unsigned": true
+		},
+		{
+			"#": "19",
+			"label": "Temperature Report Hysteresis",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "20",
+			"label": "Meter Report Interval",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 65535,
+			"defaultValue": 840,
+			"unit": "seconds",
+			"unsigned": true
+		},
+		{
+			"#": "21",
+			"label": "Action After Error",
+			"description": "Device will attempt to turn on again after timeout period",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unit": "seconds",
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Turn off (Display error)",
```

Comment:
```suggestion
					"label": "Stay off (Display error)",
```

Reply:
together with the label change above this seems to make more sense?

Reply:
agreed